pallets/asset-manager/src/weights.rs

```diff
@@ -39,124 +39,124 @@
 #![allow(unused_parens)]
 
 use frame_support::{
-	traits::Get,
-	weights::{constants::RocksDbWeight, Weight},
+    traits::Get,
+    weights::{constants::RocksDbWeight, Weight},
```

Comment:
About formatting weights file, you better change the weight template, or we have to format it while every time we re-benchmark runtime. 

https://github.com/Manta-Network/Manta/blob/manta/.github/resources/frame-weight-template.hbs#L32

Reply:
adapted it, thanks!